vanilla/README.md

```diff
@@ -0,0 +1,31 @@
+# Vanilla Autoencoder
+
+#### A Vanilla Autoencoder is an autoencoder which learns to reproduce the input from a compressed representation of the output.
```

Comment:
It shouldn't be "compressed representation of output". Change it to "the compressed representation of the output of encoder i.e. encoding" or "latent space representation of the input obtained by passing through the encoder".

Reply:
Made the suggested changes.